.github/workflows/health_checks.yml

```diff
@@ -175,7 +177,7 @@ jobs:
             # The SHA of the parent of HEAD commit on main branch.
             # This assumes linear history of main branch, i.e. one parent.
             # These events have only information about HEAD commit, hence the need for lookup.
-            baseline_commit_sha=$(gh search commits --repo ${{ github.repository }} --hash ${{ github.event.workflow_run.head_sha }} --json parents --jq '.[0].parents[0].sha')
+            baseline_commit_sha=$(gh api /repos/${{ github.repository }}/commits/${{ github.sha }} | jq -r '.parents[0].sha')
```

Comment:
I had to change this call to use rest api.
`gh search commits` only searches main branch and doesn't work on other branches.